packages/api/src/constants.js

```diff
@@ -1,10 +1,11 @@
-// let MAGIC_SECRET_KEY, SALT, PINATA_JWT, SENTRY_DSN, DATABASE_TOKEN, CLUSTER_SERVICE
+// let MAGIC_SECRET_KEY, SALT, PINATA_JWT, SENTRY_DSN, DATABASE_TOKEN, CLUSTER_SERVICE, MAILCHIMP_API_KEY
```

Comment:
Please update the README with instructions to set this secret along with the others.

Reply:
I ticketed this: https://github.com/nftstorage/nft.storage/issues/904

Reply:
readme updated for production.